### DISCUSSION TITLE
Reflections on Today's Operating Systems

### DISCUSSION THREAD

**Discussion by louies0623** [helloSystem/hello#450] [General]
"Less is more" is good in concept, and can be seen everywhere in design or living environment, but today's system and graphic design are too flat, lack of vitality or close to life, such as Skeuomorphic icons, (currently only  Favorite UI design such as Microsoft's Windows 8.1 Metro design, Google Material design, at least they still retain the visual three-dimensional and sense of light and shadow)
article:
https://designmodo.com/flat-vs-material/
https://uxplanet.org/skeuomorphic-design-a-beginners-guide-429570e9a965
https://uxplanet.org/a-look-at-skeuomorphic-ui-design-32f50016a50a
Nothing Phone CEO video https://youtu.be/13PowoqJkEM?t=393
Google Designer 
https://youtu.be/w2JUhDd0CAA

Skeuomorphic is not without disadvantages, but how to make textures and three-dimensional materials, such as presenting mac os 10.6 window UI or mixing software UI or icon, then this is a difficult process, and I don’t know how to deal with it..  .I can only imagine them using 3D modeling to render.

Then think about it, If the most extreme function of an operating system with fewer but more functions, such as Gnome removes desktop icons, theme windows, login mouse buttons, Ubuntu removes compression software,  Then the user account setting is removed, the date and time setting is removed, the keyboard input method is removed, the background image is removed, and finally even these most basic GUIs are removed, we get a "POS in the mouth of Gordon Ramsay".
[POS](https://youtu.be/Cg135Aw3gVM)
[User](https://youtu.be/6j4f0xY5AFE)
The user is not a programmer, and if it is used in a high-level work environment, it should never be a terminal command to the user.
For example, your relatives are in emergency treatment, but the medical staff are trying to solve the problem of transferring files by computer, and a lot of unnecessary time is wasted just using the terminal.

**Comment by louies0623** [comment]
What are the good things in other operating systems that we can refer to, let us think about it together.

**Reply by probonopd** [reply]
I wrote about it in my series about [#LinuxUsability](https://twitter.com/hashtag/LinuxUsability) ([part 1](https://medium.com/@probonopd/make-it-simple-linux-desktop-usability-part-1-5fa0fb369b42), [part 2](https://medium.com/@probonopd/make-it-simple-linux-desktop-usability-part-2-d34b86fd9b79), [part 3](https://medium.com/@probonopd/make-it-simple-linux-desktop-usability-part-3-780f127f5794), [part 4](https://medium.com/@probonopd/make-it-simple-linux-desktop-usability-part-4-13be6486b358), [part 5](https://medium.com/@probonopd/make-it-simple-linux-desktop-usability-part-5-d810a0d2f368), [part 6](https://medium.com/@probonopd/make-it-simple-linux-desktop-usability-part-6-1c03de7c00a9)).

**Comment by louies0623** [comment]
Although I don't like the current Windows or Mac OS system design, it doesn't mean I like Linux DE. Like MS and Apple, they will use their systems to attract customers to increase the utilization rate of their systems. Developers  Will think about the UI that can make users more convenient; but Linux DEs are not, their UI is too messy e.g KDE ,[The choice paradox]( http://youtu.be/VO6XEQIsCoM) problem caused by their UI, or because a certain developer proposed to remove important functions to make users more inconvenient, and most of them did not solve the problem for users  on problems that require regular use of the terminal.
 Android is a counterexample, is it Linux?  Logically speaking, it is, but why it is a system used by the public, because it solves the user's bad impression of using Linux DE.
